distribution/systemd/vector@.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Vector - %i
+Documentation=https://vector.dev
+After=network-online.target local-fs.target remote-fs.target time-sync.target
+Requires=network-online.target local-fs.target remote-fs.target time-sync.target
+
+[Service]
+User=vector
+Group=vector
+ExecStartPre=/usr/bin/vector validate --config-toml /etc/vector/%i.toml
+ExecStart=/usr/bin/vector --config-toml /etc/vector/%i.toml
+ExecReload=/usr/bin/vector validate --config-toml /etc/vector/%i.toml
+ExecReload=/bin/kill -HUP $MAINPID
+Restart=always
+AmbientCapabilities=CAP_NET_BIND_SERVICE
+EnvironmentFile=-/etc/default/vector
```

Comment:
Should this be per service as well?

Reply:
The `ExecReload`, `Restart` and `AmbientCapabilities` are already per service.
I could change the `EnvironmentFile` to be something like `/etc/default/vector-%i`.
What do you think?

Reply:
I have added a %i for the default value to ensure the isolation.